src/mumble/TalkingUI.cpp

```diff
@@ -752,6 +752,13 @@ void TalkingUI::on_settingsChanged() {
 		}
 	}
 
+	// Update Background Color
+	for (auto& currentContainer : m_containers) {
+		for (auto &currentEntry : currentContainer->getEntires()) {
```

Comment:
```suggestion
		for (auto &currentEntry : currentContainer->getEntries()) {
```
typo that caused the CI to fail